resources/js/Components/Form/CharacterCreateForm.tsx

```diff
@@ -55,82 +96,132 @@ const CharacterCreateForm = ({
                 {activeStep === 0 && (
                     <>
                         <Typography>
-                            Fill out the following fields with your
-                            character&apos;s details.
+                            Fill out the following fields with your character&apos;s
```

Comment:
really don't like hard coding strings like this, perhaps we can add a translation key system? will make code look much cleaner

Reply:
Laravel has an in-built translation string system in the `resources/lang` folder, i could pass each of those files as universal props if you want, but i might end up being a lot of data

Reply:
if not, we can also use `i18next`, and manage it with json

Reply:
icebox issue i guess